### ISSUE TITLE
Authenticode and .NET/MSBuild

### ISSUE THREAD

**Issue by SamuelEnglard** [dotnet/docs#8370]
The link for using Authenticode is for signing Windows drivers/C++ code. Would be nice to have documentation for signing .NET code using and/or using MSBuild to do so. Maybe not in this section but in general.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 90b392eb-c8a1-147f-e3fb-35b8ec702d53
* Version Independent ID: ffd3b865-f141-3f92-473b-fbfb9d32913b
* Content: [Strong naming and .NET libraries](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/strong-naming)
* Content Source: [docs/standard/library-guidance/strong-naming.md](https://github.com/dotnet/docs/blob/master/docs/standard/library-guidance/strong-naming.md)
* Product: **dotnet**
* GitHub Login: @JamesNK
* Microsoft Alias: **mairaw**

**Comment by JamesNK** [comment]
Docs team: do you know of an article that discusses Authenticode in the context of .NET? It would be nice to link to something more pertinent.

// @onovotny

**Comment by mairaw** [comment]
@JamesNK these are some that I found:
- https://docs.microsoft.com/visualstudio/deployment/clickonce-and-authenticode
- https://docs.microsoft.com/dotnet/framework/unmanaged-api/authenticode/

Not sure if any of these is what you're looking for?

**Comment by JamesNK** [comment]
Not really. There is a blog post that talk about Authenticode signing and .NET but it is completely out of date:

https://blogs.msdn.microsoft.com/shawnfa/2005/12/13/authenticode-and-assemblies/

I think it is a gap in the docs.

**Comment by svick** [comment]
Does anyone actually use Authenticode for .Net assemblies? Is it usable cross-platform?

If not, maybe it shouldn't be recommended here?

**Comment by onovotny** [comment]
Lots of people Authenticode .NET assemblies. It's the only way to guarantee provenance and ensure it wasn't tampered with.

**Comment by onovotny** [comment]
Next step is to get those Authenticode tools ported to .NET Core. Which will happen.

**Comment by JamesNK** [comment]
> The fact that most developers don't is likely due to the cost of the certs.

And it is never talked about to .NET devs, or well documented.

**Comment by SamuelEnglard** [comment]
Still not documented @mairaw (I would tag the team, but I cannot)

**Comment by BillWagner** [comment]
Reopening per @SamuelEnglard comment.

**Comment by mletterle** [comment]
This is a good idea.
